cmd/snap/cmd_list_test.go

```diff
@@ -56,7 +56,7 @@ func (s *SnapSuite) TestList(c *check.C) {
 			c.Check(r.Method, check.Equals, "GET")
 			c.Check(r.URL.Path, check.Equals, "/v2/snaps")
 			c.Check(r.URL.RawQuery, check.Equals, "")
-			fmt.Fprintln(w, `{"type": "sync", "result": [{"name": "foo", "status": "active", "version": "4.2", "developer": "bar", "revision":17}]}`)
+			http.Redirect(w, r, r.URL)
```

Comment:
`cmd/snap/cmd_list_test.go:59: not enough arguments in call to http.Redirect`

Missing status at the end (probably `http.StatusFound`).